src/containers/MyNdla/components/Toolbar.tsx

```diff
@@ -73,25 +75,51 @@ const StyledPageContent = styled(PageContent, {
 });
 
 interface Props {
-  buttons?: ReactNode;
-  dropDownMenu?: ReactNode;
+  menuItems?: MenuItemProps[];
   showButtons?: boolean;
 }
 
-const Toolbar = ({ buttons, dropDownMenu, showButtons }: Props) => {
+const Toolbar = ({ menuItems, showButtons }: Props) => {
+  const { t } = useTranslation();
   const { user } = useContext(AuthContext);
+
   return (
-    <ToolbarContainer data-visible={!!buttons || !!dropDownMenu || !!user?.arenaEnabled}>
+    <ToolbarContainer data-visible={!!menuItems?.length && !!user?.arenaEnabled}>
```

Comment:
Burde vel være "||" her?

Reply:
Oppdaterte begge to i forbindelse med den andre kommentaren din 😅